docs/js/datasource.js

```diff
@@ -0,0 +1,52 @@
+$(function () {
+    // let's hit the JSON endpoint
+    var containerDivId = "datasource-table";
+    $.ajax({
+        dataType: "json",
+        url: "http://housinginsights.us-east-1.elasticbeanstalk.com/api/meta",
+        success: function (data) {
+            // This is the ID (accessed from a hidden div in the template)
+            var id = document.getElementById(containerDivId).childNodes[0].id;
+            if (!data[id]) {
+                console.log("There is no data for this datasource! Please add some.");
+                return false;
+            }
+
+            // from https://stackoverflow.com/questions/15164655/generate-html-table-from-2d-javascript-array
```

Comment:
@marwahaha for future reference, if you do any work within the tool itself, I made a "reusable chart" that turns json into table form in `/docs/tool/js/charts/D3Table.js`. No need to change this now of course.